rspec-core/lib/rspec/core/example_group.rb

```diff
@@ -344,8 +344,14 @@ def self.define_nested_shared_group_method(new_name, report_label="behaves like"
       # context.
       #
       # @see SharedExampleGroup
-      def self.include_context(name, *args, &block)
-        find_and_eval_shared("context", name, caller.first, *args, &block)
+      def self.include_context(name, *args)
+        if block_given?
+          raise ArgumentError,
+                "Don't pass a block to `include_context`. " \
+                "Use `it_behaves_like` instead, or place the block content " \
+                "after the statement."
```

Comment:
`it_behaves_like` is not a proper replacement for a correctly semantically used `include_context`, should we instead just
```suggestion
                "Don't pass a block to `include_context`. " \
                "Place the block content after the statement."
```

Reply:
No because sometimes you actually want `it_behaves_like` and the usage was just broken, see suggestion below.